### PR TITLE
Nbo/radio group component prod 10498

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import React from 'react';
+import { BasicRadioInput } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { TranslationUtils } from '../../../../utils';
+
+export const GenericRadioInput = ({ parameterData, parametersState, setParametersState, context }) => {
+  const { t } = useTranslation();
+  let enumValues = parameterData.enumValues;
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js:12]: Starting with v5., enum values will no longer get retrieved from `parameterData.enumValues` but from `parameterData.options.enumValues` instead. You can use this instead:
`let enumValues = ConfigUtils.getParameterAttribute(parameterData, 'enumValues');`

See _src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericEnumInput.js_ for an example

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const textFieldProps = {
+    disabled: !context.editMode,
+    id: parameterData.id,
+  };
+
+  if (!enumValues) {
+    console.warn(
+      `Enum values are not defined for scenario parameter "${parameterData.id}".\n` +
+        'Please provide an array in the "enumValues" field for this parameter in the parameters configuration file.'
+    );
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js:22]: The "enumValues" field must be replaced by `options.enumValues`

**Review by csm-thu — CHANGES_REQUESTED** [review]

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js (hunk 1 above)
Comment (nborde-CSM) [reply src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericRadioInput.js:12]: Should I do this modification in others generic inputs ?

**Review by esasova — APPROVED** [review]
Just one typo, LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by csm-thu — APPROVED** [review]

**Review by csm-thu — APPROVED** [review]

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**